llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -979,11 +986,11 @@ def Feature32Bit
 def Feature64Bit
     : SubtargetFeature<"64bit", "IsRV64", "true", "Implements RV64">;
 def IsRV64 : Predicate<"Subtarget->is64Bit()">,
-                       AssemblerPredicate<(all_of Feature64Bit),
-                                         "RV64I Base Instruction Set">;
+             AssemblerPredicate<(all_of Feature64Bit),
+                               "RV64I Base Instruction Set">;
```

Comment:
Is this line supposed to have two more spaces to be aligned with the start of `all_of`?

Reply:
edit: nevermind it looks good

Reply:
edit: i mean the `(`